src/screens/CreatePost/elements/SheetPrivacy.js

```diff
@@ -4,54 +4,54 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
+import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
-import {COLORS} from '../../../utils/theme';
 
 const SheetPrivacy = ({privacyRef, data, select, onSelect}) => (
-  <RBSheet
-    ref={privacyRef}
-    closeOnDragDown={true}
-    closeOnPressMask={true}
-    customStyles={{
-      container: styles.containerSheet,
-      draggableIcon: styles.draggableIcon
-    }}>
-    <View style={styles.container}>
-      <Text style={styles.header}>Privacy</Text>
-      {data.map((value, index) => (
-        <FlatListItem
-          key={index}
-          value={value.label}
-          index={index}
-          select={select}
-          onSelect={onSelect}
-          icon={value.icon}
-          desc={value.desc}
-        />
-      ))}
-    </View>
-  </RBSheet>
-);
+    <RBSheet
+      ref={privacyRef}
+      closeOnDragDown={true}
+      closeOnPressMask={true}
+      customStyles={{
+        container: styles.containerSheet,
+        draggableIcon: styles.draggableIcon,
+      }}>
+      <View style={styles.container}>
+        <Text style={styles.header}>Privacy</Text>
+        {data.map((value, index) => (
+          <FlatListItem
+            key={index}
+            value={value.label}
+            index={index}
+            select={select}
+            onSelect={onSelect}
+            icon={value.icon}
+            desc={value.desc}
+          />
+        ))}
+      </View>
+    </RBSheet>
+  );
 
 export default SheetPrivacy;
 
 const styles = StyleSheet.create({
   container: {
     paddingTop: 20,
-    paddingBottom: 38
+    paddingBottom: 38,
   },
   header: {
-    color: COLORS.black,
+    color: colors.black,
     fontFamily: fonts.inter[600],
     fontSize: 18,
     fontWeight: 'bold',
-    paddingHorizontal: 20
+    paddingHorizontal: 20,
   },
   containerSheet: {
     borderTopRightRadius: 20,
-    borderTopLeftRadius: 20
+    borderTopLeftRadius: 20,
   },
   draggableIcon: {
-    backgroundColor: COLORS.alto
-  }
+    backgroundColor: colors.alto,
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the styles are consistent and follow the new color scheme. However, it's important to ensure that these color changes don't affect the accessibility of the application. For example, make sure there is enough contrast between text and background colors for readability.
```diff
-    color: COLORS.black,
+    color: colors.black,
```
```diff
-    backgroundColor: COLORS.alto
+    backgroundColor: colors.alto,
```
Please verify that the new colors still meet accessibility standards.


<!-- This is an auto-generated comment by OSS CodeRabbit -->